lib/internal/Magento/Framework/Escaper.php

```diff
@@ -71,7 +71,7 @@ function ($errorNumber, $errorString) {
                         throw new \Exception($errorString, $errorNumber);
                     }
                 );
-                $string = mb_convert_encoding($data, 'HTML-ENTITIES', 'UTF-8');
+                $string = mb_convert_encoding(''.$data, 'HTML-ENTITIES', 'UTF-8');
```

Comment:
It's better to use `(string)` cast as more obvious

Reply:
fixed it